checker/tests/resourceleak/SwitchExpressions.java

```diff
@@ -0,0 +1,180 @@
+// @below-java17-jdk-skip-test
```

Comment:
These tests are exactly the same as those in `TernaryExpressions`, but using switch expressions instead